aries_cloudagent/multitenant/admin/routes.py

```diff
@@ -1,8 +1,14 @@
 """Multitenant admin routes."""
-
+from aries_cloudagent.wallet.error import WalletSettingsError
```

Comment:
This is a package-local import, it should be with the other relative imports below.

Reply:
Fixed